docs/source/data_quality.rst

```diff
@@ -1,5 +1,9 @@
 Data Quality
 ============
 
-Data quality checks are run after the data are paired or on-demand by selecting rows in the inventory
-page and clicking the action button.
+Data quality checks are run after the data are paired, during import of Properties/TaxLots, or on-demand by selecting rows in the inventory
```

Comment:
@RDmitchell in reference to your question. https://github.com/SEED-platform/seed/pull/1877#issuecomment-491019562
This elaborates what I found to be true in the app. Let me know if I misrepresented anything or got anything wrong, and I'd be happy to update this document and PR as needed.